.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -35,16 +39,33 @@ Please check followings before submitting a new issue.
 * Step2
 * etc...
 
+## Is the problem still there under "Safe mode"?
+
+<!--
+https://hexo.io/docs/commands#Safe-mode
+
+"Safe mode" will disable all the plugins and scripts.
+If your problem disappear under "Safe mode" means the problem is probably at your newly installed plugins, not at hexo.
+-->
+
 ## Environment & Settings
```

Comment:
As you know. A while ago one of the hexo user submit his/her secret server information. 
I want to add a message just like `PLEASE DO NOT SUBMIT SECRET INFORMATION`.

Reply:
`Please DO NOT CONTAINS SECRET INFORMATION (likes password or GitHub Personal Access Token) when submit a new issue`.

Is that ok?

Reply:
Added.